src/icons/icon-success-vivo-new.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {useTheme} from '../hooks';
+import {getAnimateCircleScaleInProps, getAnimateDrawLineProps} from '../utils/animation';
+
+type Props = {
+    size?: number | string;
+};
+
+const IconSuccessVivoNew = ({size = 48}: Props): JSX.Element => {
+    const {platformOverrides} = useTheme();
+
+    return (
+        <svg width={size} height={size} viewBox="0 0 64 64" fill="none">
+            <circle
+                cx="32"
+                cy="32"
+                r="24.9375"
+                stroke="url(#vivoNewGradient)"
```

Comment:
hmmm... can't this be problematic if we have multiple instances of the component rendered in the same page?
I mean, you'd have repeated ids

Reply:
Discussed offline. We will use `useAriaId` function to generate an unique id for the gradient